cmd/wallet-web/src/pages/mixins/common/helper.js

```diff
@@ -89,6 +89,50 @@ export const isVPType = (type) => toLower(type) == 'verifiablepresentation';
 export const getCredentialType = (types) =>
   types.filter((type) => type != 'VerifiableCredential')[0];
 
+// function to replace json path from the json objects
+export function getCredentialDisplayData(vc, manifest) {
+  // title
+  const titleFound = findByPath(vc, manifest.title.path);
+  const title = titleFound ? titleFound : manifest.title.fallback;
+
+  // icon
+  const icon = manifest.icon;
+
+  // find properties
+  const _readProperty = (property) => {
+    const pKey = property[0];
+    const vals = findByPath(vc, property[1].path);
+    const value = vals ? vals : '';
+    const label = property.label;
+    const type = property.type;
+    const format = property.format;
+    return {
+      [pKey]: {
+        label,
+        value,
+        type,
+        format,
+      },
+    };
+  };
+
+  const properties = Object.entries(manifest.properties).map(_readProperty);
```

Comment:
Need a default logic for credential type not found in manifest, you need a default manifest entry. and in those cases you have to display all properties.

Reply:
Created #1013 to address